### ISSUE TITLE
Packer 0.8.1 seems to break nubis-builder

### ISSUE THREAD

**Issue by limed** [Nubisproject/nubis-builder#69]
After upgrading packer from 0.7.5 -> 0.8.1 nubis-builder doesn't seem to build anymore. The error I'm getting can be references here[1]

[1]: https://gist.github.com/limed/fd93de86169c0bc65eb6

**Comment by gozer** [comment]
==> amazon-ebs-amazon-linux: Error launching source instance: InvalidBlockDeviceMapping: Volume of size 0GB is smaller than  snapshot 'snap-22a28b61', expect size >= 8GB
==> amazon-ebs-amazon-linux: 	status code: 400, request id: []
==> amazon-ebs-amazon-linux: No AMIs to cleanup
==> amazon-ebs-amazon-linux: Deleting temporary security group...
==> amazon-ebs-amazon-linux: Deleting temporary keypair...
Build 'amazon-ebs-amazon-linux' errored: Error launching source instance: InvalidBlockDeviceMapping: Volume of size 0GB is smaller than  snapshot 'snap-22a28b61', expect size >= 8GB
	status code: 400, request id: []
 
==> Some builds didn't complete successfully and had errors:
--> amazon-ebs-amazon-linux: Error launching source instance: InvalidBlockDeviceMapping: Volume of size 0GB is smaller than  snapshot 'snap-22a28b61', expect size >= 8GB
	status code: 400, request id: []

**Comment by gozer** [comment]
This is fallout from the workaround we did for mitchellh/packer#1783

Looks like the problem was finally fixed, so the workaround isn't needed for 0.8.1 anymore. However, this is still an issue until we can upgrade wholesale.

In the meantime, I believe just adding "volume_size" to our launch_block_device_mappings should do the trick, i.e.: 

```diff
     "launch_block_device_mappings": [{
       "device_name": "/dev/sda1",
       "volume_type": "gp2",
+      "volume_size": "8",
```
Can you try and see if that works?
